packages/backend/src/peripherals/database/migrations/023_tx_count.ts

```diff
@@ -0,0 +1,27 @@
+/*
+                      ====== IMPORTANT NOTICE ======
+
+DO NOT EDIT OR RENAME THIS FILE
+
+This is a migration file. Once created the file should not be renamed or edited,
+because migrations are only run once on the production server. 
+
+If you find that something was incorrectly set up in the `up` function you
+should create a new migration file that fixes the issue.
+
+*/
+
+import { Knex } from 'knex'
+
+export async function up(knex: Knex) {
+  await knex.schema.createTable('tx_count', function (table) {
```

Comment:
This PR introduces `tx` as a shorthand for `tx` but in general in we use `transaction` everywhere. Because we dont use other shorthands too I think it's worthwile to maintain the longform. This applies to all files in this PR.